lib/utils/logger.js

```diff
@@ -247,7 +289,8 @@ const logger = {
 	 * | ['array', any[]]
 	 * | ['element', ElementStruct]
 	 * | ['function', string]
-	 * | ['NaN']
+	 * | ['number', 'NaN' | 'Infinity' | '-Infinity']
```

Comment:
So second item in array can be only  'NaN' | 'Infinity' | '-Infinity' but not a number?